action.yml

```diff
@@ -0,0 +1,9 @@
+  name: 'Publishing Parasoft SOAtest results'
+  description: 'A GitHub Action for publishing Parasoft SOAtest results to GitHub in Junit.'
```

Comment:
Please changes to:
A GitHub Action for collecting functional tests results with Parasoft SOAtest and publish results directly in GitHub.